src/orion/plotting/backend_plotly.py

```diff
@@ -143,6 +145,130 @@ def get_dimension(data, name, dim):
     return fig
 
 
+def rankings(experiments, order_by, **kwargs):
+    """Plotly implementation of `orion.plotting.rankings`"""
+
+    def reformat_competitions(experiments):
+        if isinstance(experiments, dict) and isinstance(
+            next(iter(experiments.values())), Iterable
+        ):
+            competitions = []
+            remaining = True
+            i = 0
+            n_competitions = len(next(iter(experiments.values())))
+            for ith_competition in range(n_competitions):
+                competition = {}
+                for name in experiments.keys():
+                    competition[name] = experiments[name][i]
+                competitions.append(competition)
+        elif isinstance(experiments, dict):
+            competitions = experiments
+        elif isinstance(experiments, Iterable) and not isinstance(experiments[0], dict):
+            competitions = {
+                f"{experiment.name}-v{experiment.version}": experiment
+                for experiment in experiments
+            }
+        else:
+            competitions = experiments
```

Comment:
looks we can combine this `else` with above `elif isinstance(experiments, dict)`?

Reply:
The `dict` is iterable, so if we remove the first elif it will enter the second elif. I don't like the look of this if-elif-else clause, but I don't see an elegant way to get around it. :(

Reply:
seems we are treating `dict of list`, `list `and `dict `finally as dict, leave `list of dict `as the other. 
how about changing the `3rd elif` to 
```
elif isinstance(experiments, list) and not isinstance(experiments[0], dict):
```

Reply:
Perhaps `elif isinstance(experiments, (list, tuple)) and not isinstance(experiments[0], dict):` would fit most use-cases.